script.js

```diff
@@ -1,4 +1,4 @@
-// -------------------- MOBILE MENU --------------------
+/ -------------------- MOBILE MENU --------------------
```

Comment:
This change introduces an invalid comment, which should result in an error being thrown. It's important to try to avoid checking in code that causes errors into git and sharing it with others. My recommendation is to step through all the changes you've made using `git diff`, `git add -p` or the diff view in VSCode, before making a commit.